src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -108,6 +122,9 @@ seastar::future<> ClientRequest::with_pg_int(Ref<PG> pgref)
   if (m->finish_decode()) {
     m->clear_payload();
   }
+  if (!m->get_hobj().get_key().empty()) {
```

Comment:
Worth comments:
"There are no users of locator. It was used to ensure that multipart-upload parts would end up in the same PG so that they could be clone_range'd into the same object via librados, but that's not how multipart upload works anymore and we no longer support clone_range via librados."

Reply:
will do.